src/signals/incident/components/form/MapSelectors/Asset/AssetSelect.tsx

```diff
@@ -13,6 +13,8 @@ import {
 
 import type { Incident, Location } from 'types/incident'
 import type { LatLngLiteral } from 'leaflet'
+import defaultFeatureUrl from 'shared/images/featureDefault.svg?url'
+import unknownFeatureUrl from 'shared/images/featureUnknown.svg?url'
```

Comment:
Can we have a standard for naming the files and variables? In some places the term 'icon' is used. Here it is 'feature'.

Reply:
To me icon was a bit confusing, because it has a more general meaning than feature. But I agree that it is not consistent, so I will change it.

Reply:
Could change it to `unknownFeatureIconUrl`?

Reply:
And name the file unknownFeautureIcon? or icon-unknown-feature ?

Reply:
There are definitely inconsistencies in the naming of files and folders across the project. I'm a proponent of camel-case naming for everything; files, folders and variables where we would use UpperCamelCase for component files and component folders. lowerCamelCase for non-component file names and variable names.

Reply:
👍